### ISSUE TITLE
Incorrect case in 6.5 boolean logic

### ISSUE THREAD

**Issue by Bolik** [lhncbc/fhirpath.js#77]
```
  console.log fhirpath.test.js:53
    6.5_boolean_logic.yaml ok1 xor emp * xor (true xor {}) 
    ERROR Error: expect(received).toEqual(expected)
    
    Expected value to equal:
      [true]
    Received:
      []
    
    Difference:
    
    - Expected
    + Received
    
    - Array [
    -   true,
    - ]
    + Array [] 
     [ '{"ok1":true,"ok2":true,"ups1":false,"ups2":false,"emp":[],"coll":[{"a":"a","b":"b"},{"a":"b","b":"a"},{"a":"c","b":"d"}]}',
      'ok1 xor emp' ]
```
But in spec {} xor true must returns empty collection
http://hl7.org/fhirpath/2018Sep/#xor

**Comment by plynchnlm** [comment]
Thanks for reporting this.   I've opened a pull request.  https://github.com/lhncbc/fhirpath.js/pull/78

**Comment by plynchnlm** [comment]
Fixed in 0.10.2.
